src/index.d.ts

```diff
@@ -18,6 +18,7 @@ export interface PopoverState {
 export type ContentRenderer = (popoverState: PopoverState) => JSX.Element;
 export type ContentLocationGetter = (popoverState: PopoverState) => ContentLocation;
 
+export type PopoverPositionBase = 'left' | 'right' | 'top' | 'bottom';
```

Comment:
This typing is used for cases where you cannot use 'custom' as a valid position value. Such as in `popoverRectForPosition`.